src/dymaptic.GeoBlazor.Core/Scripts/dotNetBuilder.ts

```diff
@@ -733,7 +733,7 @@ export function buildDotNetBookmark(bookmark: any): DotNetBookmark {
 }
 
 export function buildDotNetViewpoint(viewpoint: any): DotNetViewpoint | null {
-    if (viewpoint === null) return null;
+    if (! hasValue(viewpoint)) return null;
```

Comment:
is the space after `!` intentional? I could see it might be useful for readability, on the other hand, it feels non-standard to me, and I wouldn't have been suprised if it didn't function correctly.

Reply:
it was a copy paste thing I think